src/middleware/shareArticle.js

```diff
@@ -0,0 +1,27 @@
+import open from 'open';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const { APP_FRONTEND_URL } = process.env.APP_FRONTEND_URL;
+const twitterShareURL = `https://twitter.com/intent/tweet?text=${APP_FRONTEND_URL}/articles/`;
```

Comment:
This is your new feature, right? Why are you pushing it in this PR?

Reply:
yes, was already started working on it but removed it, thanks